README.md

```diff
@@ -19,12 +19,25 @@ Clone the repository
 git clone https://github.com/greypants/gulp-rails-pipeline.git
 ```
 
-Enter into the directory and run bundle install
+Enter into the directory and run bundle install.
 ```
 cd gulp-rails-pipeline
 bundle install
 ```
 
+If you are on mac you may be required to run the following to [install PostgreSQL libraries successfully](http://stackoverflow.com/a/28206009/1376627).
```

Comment:
To be honest I'm not sure why I had to install it in this way or why I needed the `pg` gem. Could be a mac thing. I'll look into it further and update it accordingly.